models/affiliation_model.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+const diffHistory = require("mongoose-diff-history/diffHistory");
+
+const { Schema } = mongoose;
+
+const affiliationSchema = new Schema(
+  {
+    // ambassadorName: {
+    //   type: String,
+    //   required: true,
+    // },
+    // ambassadorId: {
+    //   type: mongoose.Schema.Types.ObjectId,
+    //   ref: "Ambassador",
+    //   required: true,
+    // },
+    affiliateCode: {
+      type: String,
+      required: true,
+    },
+    affiliateLink: {
+      type: String,
+      required: true,
+    },
```

Comment:
@dev-scott I think the affiliateCode alone is sufficient, with it we shall be able to contruct any link we want or the ambassador shall want.

So you can remove ```affiliateLink```